bot/controllers/channels.js

```diff
@@ -0,0 +1,111 @@
+module.exports = () => {
+  const util = require('apex-util');
+
+  const _run = (message) => {
+    const ctrls = [
+      {
+        cmd: '!channels',
+        example: '!channels',
+        title: 'List All Channels',
+        desc: 'List all Armada Channels',
+        showWithHelp: true,
+        posTargetUser: null,
+        posSecondaryCmd: null,
+        regexSplitCharacter: null,
+        allowInDM: false,
+        resType: 'dm',
+        action: (message) => {
+          const channels = [];
+          util.log('help!', message);
```

Comment:
This log seems out of place. 

I suggest it's removal or enhancement to provide more contextual information with the title.